phpseclib/Net/SFTP.php

```diff
@@ -980,11 +952,9 @@ private function readlist($dir, $raw = true)
      *
      * Intended for use with uasort()
      *
-     * @param array $a
-     * @param array $b
      * @return int
      */
-    private function comparator($a, $b)
+    private function comparator(array $a, array $b)
```

Comment:
In reviewing the code it doesn't look like this returns a mixed type - looks like it just returns an int so we should prob add that as a return type hint.

Reply:
Done